### PR TITLE
Make sure that clock app appears on unity8 shell.

### PR DESCRIPTION
**Pull request by renatofilho**
- Update desktop file with the correct values 
- Added qtorganizer5-eds as dep to get alarms working

### CODE REVIEW

File: ubuntu-clock-app/setup/gui/ubuntu-clock-app.desktop [hunk 1]
```diff
@@ -132,7 +132,7 @@ Keywords[uk]=time;alarm;alert;clock;world;stopwatch;час;нагадуванн
 Keywords[zh_CN]=time;alarm;alert;clock;world;stopwatch; 时间;闹铃;闹钟;世界时;秒表;
 Keywords[zh_TW]=time;alarm;alert;clock;world;stopwatch;時鐘;鬧鐘;鬧鈴;世界;碼表;
 Exec=ubuntu-clock-app
-Icon=${SNAP}/meta/gui/clock-app@30.png
```
Comment (dholbach) [inline ubuntu-clock-app/setup/gui/ubuntu-clock-app.desktop]: Could we still use the ${SNAP} variable?

File: ubuntu-clock-app/snapcraft.yaml [hunk 2]
```diff
@@ -39,10 +40,13 @@ parts:
       - qtubuntu-desktop
       - qml-module-qtsysteminfo
       - qtmir-desktop
+      - qtorganizer5-eds
       - mir-graphics-drivers-desktop
     snap:
       - -usr/share/doc
       - -usr/include
+    after:
+      - desktop-gtk3
```
Comment (dholbach) [inline ubuntu-clock-app/snapcraft.yaml]: Why `gtk3` and not `qt5`?

Comment (renatofilho) [inline ubuntu-clock-app/snapcraft.yaml]: We need glib/gsettings paths and some environment initializations that is done by this launcher. This is necessary to communicate with EDS.

File: ubuntu-clock-app/setup/gui/ubuntu-clock-app.desktop (hunk 1 above)
Comment (renatofilho) [inline ubuntu-clock-app/setup/gui/ubuntu-clock-app.desktop]: No, We need the real path of the icon. This desktop file is read from unity8 outside the SNAP enviroment.

File: ubuntu-clock-app/snapcraft.yaml (hunk 2 above)
Comment (dholbach) [inline ubuntu-clock-app/snapcraft.yaml]: Oh ok.

**Comment by dholbach** [comment]
Thanks for the update, merging!

**Pull request merged**